### PR TITLE
Add missing translation.

### PR DESCRIPTION
**Pull request by liijunwei**
+ [Ayx03](https://github.com/Ayx03)发现了翻译缺失 https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/pull/149#pullrequestreview-902514992
+ [original](http://www.catb.org/~esr/faqs/smart-questions.html)
> If you still didn't find anything, post your question on the one site where it's most on-topic. Use the formatting tools, especially for code, and add tags that are related to the substance of your question (particularly the name of the programming language, operating system, or library you're having trouble with). If a commenter asks you for more information, edit your main post to include it. If any answer is helpful, click the up arrow to upvote it; if an answer gives a solution to your problem, click the check under the voting arrows to accept it as correct.

### CODE REVIEW

**Comment by liijunwei** [comment]
繁体中文 为google translate的结果，表达方式可能有问题， 请指出或者帮忙修改😅

File: README-zh_CN.md [hunk 1]
```diff
@@ -154,6 +154,8 @@ Copyleft 2001 by D.H.Grand(nOBODY/Ginux), 2010 by Gasolin, 2015 by Ryan Wu
 
 因为 Google 索引是即时的，在看 Stack Exchange 之前先在 Google 搜索。有很高的几率某人已经问了一个类似的问题，而且 Stack Exchange 网站们往往会是搜索结果中最前面几个。如果你在 Google 上没有找到任何答案，你再到特定相关主题的网站去找。用标签（Tag）搜索能让你更缩小你的搜索结果。
 
+如果你仍然没有找到任何线索，请将你的问题发布在最贴合主题的网站上。把问题描述清楚，把代码格式调好，然后添加与你问题内容相关的标签(特别是你遇到问题的编程语言、操作系统或库的名称)。如果评论者要求你提供更多信息，请在你的帖子里补充这些信息。 如果答案有帮助，请点击向上箭头为它点赞；如果答案解决了你的问题，请点击投票箭头下的复选框把它设为问题的答案。
```
Comment (kang8) [inline README-zh_CN.md:157]:
```suggestion
如果你仍然没有找到任何线索，请将你的问题发布在最贴合主题的网站上。把问题描述清楚，把代码格式调好，然后添加与你问题内容相关的标签(如你遇到问题的编程语言、操作系统或库的名称)。如果评论者要求你提供更多信息，请在你的帖子里一并补充。 如果答案有帮助，请点击向上箭头为它点赞；如果答案解决了你的问题，请点击投票箭头下的复选框把它设为问题的答案。
```

**Review by kang8 — CHANGES_REQUESTED** [review]
翻译优化，一些建议

File: README-zh_CN.md (hunk 1 above)
Comment (liijunwei) [reply README-zh_CN.md:157]: 前面的"particularly"我建议保留它的含义；后面的改动我认为很合理, 我改一下
谢谢你 @kang8

**Review by kang8 — APPROVED** [review]

**Pull request closed without merging**